task-3/routers/user.ts

```diff
@@ -0,0 +1,77 @@
+import express, { Request } from "express";
+import { ValidatedRequest } from "express-joi-validation";
+
+import { UserService } from "../services/user";
+import {
+  validator,
+  postSchema,
+  putSchema,
+  CreateUserSchema,
+} from "../validation";
+
+const router = express.Router();
+
+router.get(
+  "/suggested",
+  async (
+    req: Request<unknown, unknown, unknown, { login: string; limit: number }>,
```

Comment:
Consider moving types either to the top of the file or to a separate file entirely.